pkg/policy/snapshots.go

```diff
@@ -0,0 +1,236 @@
+package policy
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/jeremywohl/flatten"
+)
+
```

Comment:
I see no reason for all snapshot functions to be part of executor as they call only methods inside here, also, I think it can be function i.e ImportSnapshot and ExportSnapshot. moreover this can be a database thing, as maybe other future parts would require to import export database data for testing.